### ISSUE TITLE
When a single file is open, open new files in same window

### ISSUE THREAD

**Issue by makzsslm** [zed-industries/zed#9370]
### Check for existing issues

- [X] Completed

### Describe the feature

**Current behavior:** When any file in open in Zed, if a new file is opened from the Finder, a new window is created.

**Desired behavior:** When any file in open in Zed, if a new file is opened from the Finder, it is added to the existing window as a new tab.

This is useful when working on separate files, as opposed to working on projects.

**Comment by xthezealot** [comment]
Weird. The behavior of my Zed is exactly the opposite: single files are opened in the last workspace used.

Personally, I don't want this to happen, as single files are often unrelated to any project already open. 

Is there a configuration key for this?
For comparison, in VS Code it's “window.openFilesInNewWindow”.

**Comment by pjv** [comment]
> Weird. The behavior of my Zed is exactly the opposite: single files are opened in the last workspace used.

Me too and this is a horrible UX if you use spaces (multiple desktops) on macos because opening a file from the finder in Zed will jarringly blast you off to another space where a Zed window is open to create a new tab for the finder-opened document. 

It would be excellent if Zed could duplicate the VSCode option and functionality of `window.openFilesInNewWindow`

**Comment by Ktoks** [comment]
> It would be excellent if Zed could duplicate the VSCode option and functionality of `window.openFilesInNewWindow`

I would appreciate this greatly.

This is the main reason my team hasn't begun using Zed - we're forced to use WinSCP for our remote access - and it would be nice to have each file open up in my main editor.

**Comment by internettechnik** [comment]
Please Open new files (e.g. when selection/clicking in the Finder) in a NEW Zed window. 

Otherwise single, unrelated files are opened within an (arbitrary) existing project window ... which is more than confusing.

**Comment by mattuylee** [comment]
yes. It's greatly appreciated if we can config Zed to open single files in a new window!

**Comment by mgsloan** [comment]
I believe that https://github.com/zed-industries/zed/pull/23726 will most of the comments on this issue, and it seems like most people were seeing the "open in existing workspace" behavior.  Since this issue is fairly old, closing it.  If the behavior here is still not working well after the next release please open a new issue.

**Comment by bluemix** [comment]
Still having the same problem in Linux

**Comment by mckrlx** [comment]
If you are on macOS, I found a workaround (with Gemini's help):

1. Open Automator and select New Document > Application.
2. Search for the "Run Shell Script" action and drag it into the workflow.
3. Set "Pass input" to as arguments.
4. Paste this into the box:  
    ```
    /usr/local/bin/zed -n "$@"
    ```
6. Save this as "Zed (New Window).app" in your Applications folder.
7. Now you can right-click a file in Finder, choose Open With > Other..., and select this new app.

**Comment by imzza** [comment]
Can we configure zed to open files in same window instead of opening in new window?

**Comment by aakhmetz** [comment]
probably it's an old issue and need to be closed, but the problem is still there
